distributed/tests/test_steal.py

```diff
@@ -972,16 +972,19 @@ async def test_lose_task(c, s, a, b):
 
 
 @pytest.mark.parametrize("interval, expected", [(None, 100), ("500ms", 500), (2, 2)])
-@gen_cluster(nthreads=[])
+@gen_cluster(nthreads=[], config={"distributed.scheduler.work-stealing": False})
 async def test_parse_stealing_interval(s, interval, expected):
```

Comment:
I suspect the new implementation is what this test is supposed to be doing. This does not take into account some weirdness around how `WorkStealing` is added as a plugin in the first place, but that's out-of-scope.
cc @fjetter